fdbcli/fdbcli.actor.cpp

```diff
@@ -2551,6 +2551,13 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 		validOptions = options->getValidOptions();
 	}
 
+	try {
+		wait(waitOrError((checkDataDistributionStatus(db, true)), delay(5)));
```

Comment:
I made it interruptible but I am hesitate to combine this with the `checkStatus` since the latter is optional, and I want to make when the CLI connecting to a cluster with DD partially/fully disabled, user will always get a fat warning.

Reply:
The purpose of being able to disable status is to be able to bring up the cli without having to wait if the cluster is down. I think we'd want to not lose that ability.

Reply:
Fair enough. Changed to be interruptable